component/admin/language/en-GB.com_jevents.ini

```diff
@@ -526,7 +526,7 @@ JEV_CALENDAR_IMPORT="Ical Import"
 JEV_TIMEZONE_INFO="Choose a valid Timezone value from <a href='http://www.php.net/manual/en/timezones.php' title='Timezone info' target='_blank'>here</a>"
 JEV_TIMEZONE="Timezone for your server<br/>(requires PHP 5.1+)"
 JEV_TIMEZONE_DESC="This is the timezone used for inputting events correctly during iCal import - this is very important when importing events from a UTC datasource or if our server is in a different DST regimen.<br/><br/>Enter the appropriate value EXACTLY as indicated via the link above.<br/><br/>This timezone is also used for determining the current time of day."
-JEV_IGNORE_EMBEDDED_CATEGORIES="Ignore embedded category names?"
+JEV_IGNORE_EMBEDDED_CATEGORIES="Replace categories with the fallback category?"
```

Comment:
But this is a yes/no field no field which says whether to use the selected category instead? 

You could have 

''Ignore imported categories and use the above selected category?'